src/sinks/gcp/chronicle_unstructured.rs

```diff
@@ -208,7 +210,10 @@ impl SinkConfig for ChronicleUnstructuredConfig {
     }
 
     fn input(&self) -> Input {
-        Input::log()
+        let requirement =
+            schema::Requirement::empty().required_meaning("timestamp", Kind::timestamp());
```

Comment:
Granted, it feels like forever ago that I looked at source log namespacing so this might not be accurate but, it kind of seems like this definition might be common in the sinks? Would it be worth extracting to a helper area in the log namespacing code that we could call, like `get_timestamp_required_meaning()` ? Or is that overkill.

Reply:
I think helper methods on `schema::Requirement` for the common 4 fields we use now could be useful. So you could write `Requirement::empty().with_timestamp()` or something similar.

Reply:
You also need to take into account optional requirements though, so the helper would either need a parameter, or 2 separate functions, so I'm not sure if it's really much easier at that point